openjdk/excludes/ProblemList_openjdk22-openj9.txt

```diff
@@ -562,9 +562,9 @@ serviceability/jvmti/RedefineClasses/RedefinePreviousVersions.java https://githu
 serviceability/jvmti/vthread/ToggleNotifyJvmtiTest/ToggleNotifyJvmtiTest.java https://github.com/adoptium/aqa-tests/issues/1297 generic-all
 serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java#default https://github.com/eclipse-openj9/openj9/issues/18426 generic-all
 serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java#no-vmcontinuations https://github.com/eclipse-openj9/openj9/issues/18426 generic-all
-serviceability/jvmti/vthread/HeapDump/VThreadInHeapDump.java https://github.com/eclipse-openj9/openj9/issues/18811 generic-all
+serviceability/jvmti/vthread/HeapDump/VThreadInHeapDump.java https://github.com/adoptium/aqa-tests/issues/1297 generic-all
```

Comment:
I'd prefer these are excluded with a link to the openj9 issues so if someone is looking to see why they are excluded, the appropriate details are easily found.

Reply:
@ShelleyLambert had mentioned that there is a test script which re-enables tests once the linked issue is closed. Using https://github.com/adoptium/aqa-tests/issues/1297 prevents the script to re-enable the test once the corresponding OpenJ9 issue is closed. @llxia Is there a way to include the OpenJ9 issue for permanently disabled tests?

Reply:
I've never seen that in action for OpenJ9 issues, so I'd be surprised if it's working. If we did use such a thing it should check for the "perm excluded" label.

Reply:
I can find examples of closed issues that have excluded tests. Here are a couple.
https://github.com/eclipse-openj9/openj9/issues/16551
https://github.com/eclipse-openj9/openj9/issues/15247

Reply:
yes, we have automation (disabled test pipeline) to auto re-enable tests based on referenced issues. However, we haven't enabled this check. For openj9, I think we should enhance the disabled test pipeline to check "perm excluded" label on the issue.

Reply:
> For openj9, I think we should enhance the disabled test pipeline to check "perm excluded" label on the issue.

I have added the **"perm excluded"** label in https://github.com/eclipse-openj9/openj9/issues/18809 and https://github.com/eclipse-openj9/openj9/issues/18811. I am closing this PR assuming that the above proposed check is implemented in the future.